### PR TITLE
Update mkldnn_quant readme

### PR DESCRIPTION
**Pull request by lidanqing-vv**
@juncaipeng 
我按要求更改了运行的参数，更新了文档，删除了两个文件cmake(新的CMakeList.txt后不需要了)
然后把 【INT8模型精度和性能结果参考】 的链接改成了指向2.0-alpha的链接

### CODE REVIEW

**Comment by lidanqing-vv** [comment]
@juncaipeng  Hi, 可以看下这个挂了的CI 是咋回事吗

**Comment by luotao1** [comment]
[PaddleModels_PaddleSlim_CI_768.log](https://github.com/PaddlePaddle/PaddleSlim/files/5943215/PaddleModels_PaddleSlim_CI_768.log)

**Comment by lidanqing-vv** [comment]
@wojtuss  Since juncai is on vocation and I can not restart PaddleSlim_CI, could you please review this PR English part?

**Review by juncaipeng — DISMISSED** [review]
lgtm

File: demo/mkldnn_quant/CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ message("flags" ${CMAKE_CXX_FLAGS})
 if(NOT DEFINED PADDLE_LIB)
   message(FATAL_ERROR "please set PADDLE_LIB with -DPADDLE_LIB=/path/paddle/lib")
 endif()
+set(DEMO_NAME sample_tester)
```
Comment (wojtuss) [inline demo/mkldnn_quant/CMakeLists.txt:12]: Should the `DEMO_NAME` be set here? If yes, the check below is redundant.

**Review by juncaipeng — APPROVED** [review]
LGTM

**Pull request merged**